typings/index.d.ts

```diff
@@ -995,6 +995,7 @@ declare module 'discord.js' {
       content?: StringResolvable,
       options?: MessageOptions | MessageAdditions | (MessageOptions & { split?: false }) | MessageAdditions,
     ): Promise<Message>;
+    public quote(content?: StringResolvable): Promise<Message>;
```

Comment:
```suggestion
    public quote(content?: string): Promise<Message>;
```

Reply:
this overload also needs to be moved ontop or below all of the reply overloads